app/services/oneoffs/npq/migrate_declarations_between_statements.rb

```diff
@@ -76,17 +92,37 @@ def each_statements_by_provider
       end
     end
 
+    def change_declaration_states_for_to_statement(to_statement, statement_line_items)
+      declarations = statement_line_items.map(&:participant_declaration).uniq
+
+      service = if to_statement.payable?
+                  ParticipantDeclarations::MarkAsPayable.new(to_statement)
+                elsif to_statement.paid?
```

Comment:
I think the paid is more tricky, first it has to go from payable to paid as there are checks in the service to only mark paid if payable. Can we avoid this all together as we shouldn't be moving declarations around if the previous statement has been paid?